src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(number);
+
+    return calculator;
+  };
```

Comment:
This is a code knowledge issue. When creating a method in an object, you don't need to use the function keyword, use the shortcut instead. So, you should replace the arrow function with a method definition like this: `operate(callback, number) {...}`.